### DISCUSSION TITLE
Option to disable console auto open

### DISCUSSION THREAD

**Discussion by Anikyte** [FakeFishGames/Barotrauma#4962] [Ideas and suggestions]
Pretty simple. An option to prevent the console from automatically opening whenever there's an error. I understand why it's designed this way, but it is kind of annoying to have the console open every 2 seconds during MP, cause someone mixed / and \. The auto popup puts mac and linux players at a significant disadvantage on modded servers, if said mods aren't perfectly cross platform compatible, and while in an ideal world this wouldn't be an issue in the first place, we don't live in such a world.

**Comment by Leonidas-ai** [comment]
Ignoring non-catastrophic errors sure would be nice

**Comment by 3e849f2e5c** [comment]
I actually went ahead and fixed _most_ if not all cases of mod incompatibility issues on Linux in https://github.com/Regalis11/Barotrauma/issues/4692, you should see the fixes included in the upcoming update. You can try out the changes on the public [unstable version](https://barotraumagame.com/announcements/barotrauma-tester-version-made-public/) right now, if there are still some cases where this happens on Linux please let me know.
